sdk/README.md

```diff
@@ -25,7 +25,7 @@ a basic sequential pipeline.
 
 2. Build the compiler:
 
-    - `pip install sdk/python`
+    - `pip install -e sdk/python`
```

Comment:
this allows the source folder to be edited and the changes are picked up immediately by the Python runtime